bin/generate

```diff
@@ -23,15 +37,16 @@ parser.on('-a', '--all', 'Generate all exercises') do
 end
 
 parser.on('--verify', 'Verify all exercises') do
-  exercises = Dir.entries('./exercises/practice').select { |f| File.directory? File.join('./exercises/practice', f) }
   exercises.each do |exercise|
     if File.exist?("./exercises/practice/#{exercise}/.meta/test_template.erb")
       current_code = File.read("./exercises/practice/#{exercise}/#{exercise}_test.rb")
       f = Tempfile.create
       Generator.new(exercise).generate(f.path)
       generated_code = f.read
-      raise RuntimeError.new("The result generated for: #{exercise}, doesnt match the current file") if current_code != generated_code
+      raise VerificationError unless current_code == generated_code
     end
+    rescue VerificationError => e
+      $stderr.puts e.message % {exercise:}
```

Comment:
ℹ️ Be aware that this syntax is relatively new, and so we should ensure that the minimum Ruby version is set to allow for this "valueless hash" syntax.

Reply:
I mean it is tooling and we have set the ruby version to 3.3 so I think we dont have to have backwards compatibility.

Reply:
It means that it is backward compatible to 3.3.0, up to 3.3.6, currently.  Locally I am running 3.3.6, and so the backward compatibility is there.